packages/dashboard/src/components/Wizard/Wizard.styles.ts

```diff
@@ -8,9 +9,10 @@ export const getWizardStyles = (): IWizardStyles => {
     },
     subwayNavSection: {
       width: '303px',
-      marginLeft: '48px',
-      marginTop: '48px',
-      marginBottom: '48px'
+      paddingLeft: '48px',
+      paddingTop: '48px',
+      paddingBottom: '48px',
+      borderRight: `1px solid ${NeutralColors.gray40}`

```

Comment:
should this use the theme color instead (i.e. theme.semanticColors.bodyDivider)

Reply:
From the [mocks](https://www.figma.com/proto/CjZhXXZ20uYf4i5yxi2H4a5m/14-M365-Admin-Guidelines-Wizards?node-id=1%3A961&scaling=min-zoom), looks like there is no border right for setup wizard?